src/components/pages/Recipients/RenderRecipientsPage.js

```diff
@@ -25,6 +25,7 @@ function RenderRecipientsPage({
   };
 
   const onCreateHousehold = householdObj => {
+    console.log('HOUSEHOLD ADDING:', householdObj);
```

Comment:
Maybe you don't want to include the console.log in production.

Reply:
Yeah, good catch